src/app/layout.tsx

```diff
@@ -1,19 +1,46 @@
-import './globals.css';
+import '@/styles/globals.css';
 
-import type { Metadata } from 'next';
-import { Inter } from 'next/font/google';
+import { MobileLayout } from '@/components/base/layouts/MobileLayout';
+import { fontInter } from '@/const/fonts/fonts';
+import { siteConfig } from '@/const/site/siteConfig';
+import { cn } from '@/lib/tailwind/utils';
 
-const inter = Inter({ subsets: ['latin'] });
+import { Metadata } from 'next';
 
 export const metadata: Metadata = {
-  title: 'Create Next App',
-  description: 'Generated by create next app',
+  title: {
+    default: siteConfig.name,
+    template: `%s - ${siteConfig.name}`,
+  },
+  description: siteConfig.description,
+  themeColor: [
+    { media: '(prefers-color-scheme: light)', color: 'white' },
+    { media: '(prefers-color-scheme: dark)', color: 'black' },
+  ],
+  icons: {
+    // TODO add actual icons
+    icon: '/favicon.ico',
+    shortcut: '/favicon-16x16.png',
+    apple: '/apple-touch-icon.png',
+  },
 };
 
-export default function RootLayout({ children }: { children: React.ReactNode }) {
+interface RootLayoutProps {
+  children: React.ReactNode;
+}
+
+export default function RootLayout({ children }: RootLayoutProps) {
   return (
-    <html lang="en">
-      <body className={inter.className}>{children}</body>
-    </html>
+    <>
+      {/* Reducing hydration error. Reference : https://nextjs.org/docs/messages/react-hydration-error */}
+      <html lang="en" suppressHydrationWarning>
+        <head />
+        <body
+          className={cn('min-h-screen bg-background font-sans antialiased', fontInter.variable)}
```

Comment:
`fontInter.variable` has no effect. It would be better to use a customized tailwind font family like:
```tsx
        <body
          className='min-h-screen bg-background font-sans antialiased font-inter'
        >
```

Reply:
By adding "fontInter.variable" to body tag's className, we can optimize fonts and it's better than using tailwind className?

I'm not 100% sure about it tho, I'll leave the reference from next/learn below.

https://nextjs.org/learn/dashboard-app/optimizing-fonts-images